pysces/vortex.py

```diff
@@ -79,6 +79,9 @@ def induced_velocity_single(self, x, xvort, gam):
         gam : float
             Strength of vortex
 
+        This method returns a vector of velocities at the points x induced
```

Comment:
I will update according to all of your preceding comments.  Thanks for the
pointers to the numpydoc and PEP 257 standards.  I absolutely agree that
standards are important and that we should adhere to them.  I'll also write
some tests for the new airfoil methods.

On Sun, Nov 22, 2015 at 10:31 AM, Clancy Rowley <notifications@github.com>
wrote:

> In pysces/vortex.py
> <https://github.com/cwrowley/pysces/pull/2#discussion_r45560186>:
>
> > @@ -79,6 +79,9 @@ def induced_velocity_single(self, x, xvort, gam):
> >          gam : float
> >              Strength of vortex
> >
> > +        This method returns a vector of velocities at the points x induced
>
> Use numpydoc
> <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>
> conventions for the returned values.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cwrowley/pysces/pull/2/files#r45560186>.
>